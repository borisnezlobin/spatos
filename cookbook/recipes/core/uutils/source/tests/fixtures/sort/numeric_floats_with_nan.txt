.03
.02
NaN
