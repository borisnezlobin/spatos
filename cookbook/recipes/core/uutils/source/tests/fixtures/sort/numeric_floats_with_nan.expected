NaN
.02
.03
